demux/cli/samplesheet.py

```diff
@@ -138,6 +141,62 @@ def get_project(project):
         # add [section] header
         click.echo('[Data]')
 
+    if application == 'nova':
+        if dualindex:
+            click.echo(click.style(f"No need to specify dual or single index for NovaSeq sample "
+                                   f"sheets, please use --shortest, --longest, or --indexlength "
+                                   f"only!", fg='red'))
+            context.abort()
+
+        lims_keys = ['fcid', 'lane', 'sample_id', 'sample_ref', 'index', 'index2', 'sample_name',
+                     'control', 'recipe', 'operator', 'project']
+        header = [Samplesheet.header_map[head] for head in lims_keys]
+
+        path = os.path.abspath(__file__)
+        dir_path = os.path.dirname(path)
+        with open(f"{dir_path}/../../files/20181012_Indices.csv") as csv_file:
+            dummy_samples_csv = csv.reader(csv_file, delimiter=',')
+            dummy_samples = [row for row in dummy_samples_csv]
+            added_dummy_samples = []
+            lanes = set([flowcell['lane'] for flowcell in raw_samplesheet])
+            flowcell_id = raw_samplesheet[0]['fcid']
+
+            for lane in lanes:
+                sample_indexes = [f"{sample['index']}" for sample in raw_samplesheet if sample['lane'] == lane]
+                for name, dummy_index in dummy_samples:
+                    if not(any(sample_index.startswith(dummy_index) for sample_index in sample_indexes)):
+                        add_dummy_sample = {'control': 'N',
+                                            'description': '',
+                                            'fcid': flowcell_id,
+                                            'index': dummy_index,
+                                            'index2': '',
+                                            'lane': lane,
+                                            'operator': 'script',
+                                            'project': 'indexcheck',
+                                            'recipe': 'R1',
+                                            'sample_id': name,
+                                            'sample_name': 'indexcheck',
+                                            'sample_ref': 'hg19'}
+
+                        added_dummy_samples.append(add_dummy_sample)
+
+            raw_samplesheet.extend(added_dummy_samples)
+
+        if indexlength:
+            raw_samplesheet = [line for line in raw_samplesheet if len(line['index'].replace('-', '')) == int(indexlength)]
+
+        for line in raw_samplesheet:
+            if '-' in line['index']:
+                index1 = line['index'].split('-')[0]
+                index2 = line['index'].split('-')[1]
```

Comment:
it is possible to do this in one line:
```suggestion
                index1, index2 = line['index'].split('-')
```

Reply:
Right, I'll fix this too.